src/cli/tc.rs

```diff
@@ -104,11 +108,37 @@ pub async fn init(file_path: &str) -> Result<()> {
 pub fn display_schema(blueprint: &Blueprint) {
   Fmt::display(Fmt::heading(&"GraphQL Schema:\n".to_string()));
   let sdl = blueprint.to_schema();
-  Fmt::display(print_schema::print_schema(sdl));
+  Fmt::display(format!("{}\n", print_schema::print_schema(sdl)));
 }
 
 fn display_config(config: &Config, n_plus_one_queries: bool) {
-  Fmt::display(Fmt::success(&"No errors found".to_string()));
   let seq = vec![Fmt::n_plus_one_data(n_plus_one_queries, config)];
   Fmt::display(Fmt::table(seq));
 }
+
+fn validate_operations(blueprint: &Blueprint, operations: Vec<String>) -> Result<()> {
+  tokio::runtime::Runtime::new()?.block_on(async {
+    let schema = blueprint.to_schema();
+    let mut execution = vec![];
+
+    for op in operations.iter() {
+      if let Ok(operation) = tokio::fs::read_to_string(op).await {
+        // Fake execution to get errors
+        let Response { errors, .. } = schema.execute(&operation).await;
```

Comment:
We don't have a way to validate errors without executing the schema?

Reply:
Well, I tried at first. But that would just be too much work and I believe it's out of the scope of the project. `async_graphql` is already parsing well, so creating an `Operation` struct to handle all that, would be a lot of duplicated efforts. But, because the resolvers are fake when checking, the schema execution doesn't return any meaning result, yet it still performs all the required checks.

Reply:
I also thought about parsing the operation then validating the fields manually, but then, it's still repetitive.